exporter/kafkaexporter/README.md

```diff
@@ -61,6 +61,8 @@ The following settings can be optionally configured:
   User should calculate this as `num_seconds * requests_per_second` where:
     - `num_seconds` is the number of seconds to buffer in case of a backend outage
     - `requests_per_second` is the average number of requests per seconds.
+- `compression` (default = none): The compression codec to compress the messages sent to Kafka. Support four compression methods: `gzip`, `snappy`, `lz4` and `zstd`.
+- `compression_level` (default = -1000): The corresponding compression level.
```

Comment:
What does the `compression_level` value mean?  Is there a scale? Is it the same for all supported compression methods? Does a lower value indicate more compression, or a higher value? What is the range of this scale?

Reply:
> What does the compression_level value mean? Is there a scale?

Yes, it is a scale. It measures the compression degree to the files.
> Is it the same for all supported compression methods?

No, each compression method has different range. For example, the valid range for gzip is 1 - 9.
> Does a lower value indicate more compression, or a higher value?

Higher value indicates more compression.
> What is the range of this scale?

The valid range for each method is as below:
gzip: 1 - 9
snappy: do not support
lz4: 1 - 17
zstd: 1 - 22

I would put all these info into README. Thanks!